src/main/scala/io/moia/scalaHttpClient/HttpClient.scala

```diff
@@ -67,7 +67,7 @@ abstract class HttpLayer[LoggingContext](
 )(implicit
     system: ActorSystem
 ) {
-  protected def sendRequest: HttpRequest => Future[HttpResponse]
+  protected def sendRequest(request: HttpRequest): Future[HttpResponse]
```

Comment:
Hm, I'm torn here.

Contra:
* I don't like that we have to provide a name now.
* `.flatMap(req => sendRequest(req))` is no nice
* technically, this is a breaking change, because overriding the `sendRequest` is part of the public interface

Pro:
* the code to override has actually gotten nicer
  https://github.com/moia-dev/scala-http-client/blob/ed230df4c0587718feffa930e83cffa6d4fc9ced/src/test/scala/io/moia/scalaHttpClient/HttpClientTest.scala#L26-L29

My proposed solution: Revert the `scapegoatVersion` and these changes, and extract that into a separate PR 😊

Reply:
Done in https://github.com/moia-dev/scala-http-client/pull/57/commits/6e975554768dcc312012eba16d903c8412933b27. I'll create the other PR when this one is merged (otherwise cherry-picking is annoying).